toolbox/oidc_verifier_test.go

```diff
@@ -53,7 +47,7 @@ func endpoint(_ context.Context, request interface{}) (response interface{}, err
 	}, nil
 }
 
-func TestGetOidcVerifier(t *testing.T) {
+/*func TestGetOidcVerifier(t *testing.T) {
```

Comment:
Is it normal that this test is commented ?

Reply:
Rewritten...